### DISCUSSION TITLE
Using playwright instead of Cypress?

### DISCUSSION THREAD

**Discussion by raed667** [ValentinH/react-easy-crop#315] [Ideas]
What do you think about moving this project to use playwright?
It will allow for better testing against multiple browsers.

-- OR

maybe also utilize https://docs.cypress.io/guides/guides/cross-browser-testing

**Comment by ValentinH** [comment]
I don't have any experience with playwright. What would be the main reasons for switching now?

Talking about e2e tests however, we discussed in the currently open PR to replace the assertions on CSS by visual regression tests. If you see what I mean 😉😉😉
